cli/command/formatter/image.go

```diff
@@ -95,21 +95,21 @@ func imageFormat(ctx ImageContext, images []types.ImageSummary, format func(subC
 			repoDigests := map[string][]string{}
 
 			for _, refString := range append(image.RepoTags) {
-				ref, err := reference.ParseNamed(refString)
+				ref, err := reference.ParseNormalizedNamed(refString)
 				if err != nil {
 					continue
 				}
 				if nt, ok := ref.(reference.NamedTagged); ok {
-					repoTags[ref.Name()] = append(repoTags[ref.Name()], nt.Tag())
+					repoTags[reference.FamiliarName(ref)] = append(repoTags[reference.FamiliarName(ref)], nt.Tag())
```

Comment:
I'd suggest breaking out `reference.FamiliarName(ref)` into a variable for readability and efficiency.